static/skin/index.js

```diff
@@ -267,35 +279,43 @@
         });
     }
 
+    function resetAllFiltersAndRedirect() {
+        window.history.pushState({}, null, `${window.location.href.split('?')[0]}?lang=`);
+        setCookie(filterCookieName, 'lang=');
+        resetAndFilter();
+        removeTagElement(false);
+        document.querySelectorAll('.filter').forEach(filter => {
+            filter.value = params.get(filter.name) || '';
+            if (filter.value) {
+                filter.style = 'background-color: #858585; color: #fff';
+            } else {
+                filter.style = 'background-color: #ffffff; color: black';
+            }
+        });
```

Comment:
Querying `params` (and subsequent logic) in a function that is supposed to reset all filters looks a little strange. However, I see that this piece of code is almost duplicated in `resetAndFilter()` and the handler installed on `window.onload`. So instead of optimizing this instance, it makes more sense to deduplicate.